bundles/org.eclipse.sirius.emfjson/src/main/java/org/eclipse/sirius/emfjson/utils/GsonEObjectDeserializer.java

```diff
@@ -401,7 +413,7 @@ private EObject loadObject(JsonObject object, boolean isTopObject) {
                 this.resource.setID(eObject, idJsonElement.getAsString());
             }
         }
-
+        this.jsonResourceProcessor.getJSonObjectBeforeLoading(eObject, object, isTopObject);
```

Comment:
Isn't that after loading since the whole behavior of the method has been executed? Why not something like `this.jsonResourceProcessor.postObjectLoading(eObject, object, isTopObject`?

Reply:
You're right, I put `beforeloading `because we are still in the `loadObject `function but all the logic have already been executed. I'll take your suggestion.